docs/accessing-services.asciidoc

```diff
@@ -178,7 +178,7 @@ spec:
         disabled: true
 ----
 
-TLS cannot be disabled for Elasticsearch.
+You cannot disable TLS for Elasticsearch.
```

Comment:
nit: this actually reads as less friendly to me, like we're stopping you from doing something rather than just a statement of fact

Reply:
Passive voice should be avoided wherever possible. But if in this case the change conveys a wrong message, then we keep the passive.